formtranslate/api.py

```diff
@@ -104,5 +104,5 @@ def form_translate(input_data, operation, version='1.0'):
     stdout, stderr = p.communicate(input_data)
     exit_code = p.wait()
 
-    return ShellResult(stdout=stdout.decode(), stderr=stderr.decode(),
+    return ShellResult(stdout=stdout.decode('utf-8'), stderr=stderr.decode('utf-8'),
```

Comment:
@czue, yes. I've added a comment to explain this PR -- how I noticed the bug, and checked I'd resolved it. 

I should also have made it clearer that the error was in the imported app (a form refered to a case property that wasn't found), not on production. I didn't test whether the issue was with stderr or stdout, but I thought it best to specify a character set for both.